examples/simple/handlers.go

```diff
@@ -47,9 +47,9 @@ func registerHTTPers(service service.Host) []uhttp.RouteHandler {
 	}
 }
 
-type simpleFilter struct{}
+type simpleMiddleware struct{}
 
-func (simpleFilter) Apply(w http.ResponseWriter, r *http.Request, next http.Handler) {
-	io.WriteString(w, "Going through simpleFilter")
+func (simpleMiddleware) Apply(w http.ResponseWriter, r *http.Request, next http.Handler) {
```

Comment:
what's the point of this being in the example if it's not used? shouldn't we change simple/main.go to use this as an option?